SolizWilson/colecciones/src/Cliente.java

```diff
@@ -0,0 +1,57 @@
+public class Cliente {
```

Comment:
please change all class and attributes names to English, it's a best practice to avoid switch between languages. For example: in the getters and setters, you have getCodigo which is a combination that is not readable.